.github/workflows/ValidatePullRequest.yml

```
@@ -5,30 +5,59 @@ name: Validate Pull Request
 on:
   pull_request:
     branches: [main, "release/**"]
-    paths-ignore:
-      - '**.md'
-      - '**.txt'
   merge_group:
 
 permissions:
   id-token: write
   contents: read
 
 jobs:
+  docs-pr:
+    runs-on: ubuntu-latest
+    outputs:
+      docs-only: ${{ steps.docs-only.outputs.result }}
+    steps:
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            docs:
+              - '**/*.md'
+              - '**/*.txt'
+            all:
+              - '**/*'
+      - uses: actions/github-script@v7
+        id: docs-only
+        with:
+          script: |
+            let docs_file_count = ${{steps.changes.outputs.docs_count}};
+            let all_file_count = ${{steps.changes.outputs.all_count}};
+            console.log(all_file_count > docs_file_count)
```

With that name, this logic also needs to change
```suggestion
            console.log(all_file_count <= docs_file_count)
```

and unfortunately so do all the "true" and "false" conditions

---

I think I'm just going to back to `console.log((all_file_count - docs_file_count) == 0)`.

---

In that case `console.log(all_file_count == docs_file_count)`

---

or triple equal for type equality as well `console.log(all_file_count === docs_file_count)`

---

I feel real dumb right now LOL